CHANGELOG.md

```diff
@@ -101,6 +101,16 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   content.
 
 * `jj git fetch -b <remote-git-branch-name>` will now warn if the branch(es)
+
+* New command `jj simplify-parents` will remove redundant parent edges.
```

Comment:
It looks like this will be inserted in the 0.22 release section, this should be moved up to the unreleased section.